src/main/kotlin/graphics/scenery/StickAndBallProteinModel.kt

```diff
@@ -0,0 +1,134 @@
+package graphics.scenery
+
+import org.joml.*
+import graphics.scenery.*
+import org.biojava.nbio.structure.*
+
+/**
+ * This is a representation of a protein structure. Displayed are the covalent bonds of the protein with atoms being
+ * represented with Icospheres and cylinders for the connections between them.
+ * @param [protein] Protein which is to be visualized.
+ *
+ * @author  Justin Buerger <burger@mpi-cbg.de>
+ */
+class StickAndBallProteinModell(protein: Protein, spaceFilling: Boolean = false,
+                                displayExternalMolecules: Boolean = false): Mesh("PrimaryStructure") {
+    val structure = protein.structure
+    private val periodicTable = PeriodicTable()
```

Comment:
The periodic table will probably be used only as one instance for all, right? If so, move to `companion object`.